apps/server/src/modules/authorization/README.md

```diff
@@ -210,18 +199,17 @@ async createUserByAdmin(userId: EntityId, params: { email: string, firstName: st
 // admin
 async editCourseByAdmin(userId: EntityId, params: { courseId: EntityId, description: string }) {
 
-    const course = this.courseService.getCourse(params.courseId);
-    const user = this.authorizationService.getUserWithPermissions(userId);
-
-   const school = course.school
-
-    this.authorizationService.hasPermissions(user, school, [Permission.INSTANCE, Permission.COURSE_EDIT]);
+   const course = this.courseService.getCourse(params.courseId);
+   const user = this.authorizationService.getUserWithPermissions(userId);
+   const school = course.school;
 
-    course.description = params.description;
+   const context = AuthorizationContextBuilder.write([Permission.INSTANCE, Permission.CREATE_USER]);
+   this.authorizationService.hasPermissions(user, school, context);
```

Comment:
```suggestion
   this.authorizationService.checkPermission(user, school, context);
```

Reply:
Allgemein find ich das mit dem vielen Code in der Doku ja nicht so gut, weil sich da sehr leicht Fehler einschleichen.

Reply:
Vielleicht dann lieber Beispiele verlinken, wo wir finden, dass es gut implementiert ist... Ist aber eher ne allgemeine Überlegung und nicht für diesen PR.

Reply:
geändert